### DISCUSSION TITLE
Label component accessibility role="label"

### DISCUSSION THREAD

**Discussion by aarongan** [radix-ui/primitives#1654] [Help]
Hi , 

I am using Label to create a label component ,and when it rendered in html, there is role="label" which is for accessibility usage been added. 
Wondering is this a valid role from Aria consortium? Since i am not finding any documentation has this role listed. if you could share will be great. thanks

eg. 
<span **role="label"** id="radix-6" class="c-hfzFYA c-hfzFYA-ilmHHQU-css">First name</span>

**Comment by andy-hook** [comment]
Hey @aarongan , [see here](https://github.com/radix-ui/primitives/issues/638#issuecomment-835241495) for rationale and discussion on this, at the bottom are some linked issues with similar questions as well.
